install.py

```diff
@@ -236,6 +236,7 @@ def build_legion(
     cuda,
     arch,
     openmp,
+    march,
```

Comment:
This and other changes in install.py and legate.mk are from another PR that this PR shouldn't include. Please clean them off.

Reply:
The rebase is somehow messed up. When I do a manual diff of the two heads (22.05 and this branch) these files don't show up as the corresponding changeset has already been merged (even before I did the rebase). I can't revert this from my branch as this would also remove it from the main branch when merged. I can create a new branch with cherry-picked changesets, but this would require a new PR as well.